### DISCUSSION TITLE
How to develop in sherpa-onnx repository

### DISCUSSION THREAD

**Discussion by thewh1teagle** [k2-fsa/sherpa-onnx#1155] [Q&A]
Every time I change something related to cmake it's fetching all the dependencies which takes a lot of time.
Can I cache them?

**Comment by csukuangfj** [comment]
please tell us what you are changing.

cmake will cache it as long as you don't delete the build directory.

please have a look at the xxx.cmake inside sherpa-onnx/cmake to find out what additional directories you can use.
